lang/main-ja.json

```diff
@@ -1,636 +1,781 @@
         "transcribing": "書き写し",
-        "unlockRoom": "",
+        "unlockRoom": "$t(lockRoomPassword)を解除する",
         "userPassword": "ユーザーのパスワード",
-        "WaitForHostMsg": "",
-        "WaitForHostMsgWOk": "",
+        "WaitForHostMsg": "ミーティングルーム <b>{{room}}</b> はまだ開始されていません。ユーザアカウントをお持ちなら、ユーザー名とパスワードを入力してミーティングを開始してください。お持ちでなければ、ミーティングが始まるまでお待ち下さい。",
+        "WaitForHostMsgWOk": "ミーティングルーム <b>{{room}}</b> はまだ開始されていません。ユーザアカウントをお持ちなら、OKを押して、ユーザー名とパスワードを入力し、ミーティングを開始してください。お持ちでなければ、ミーティングが始まるまでお待ち下さい。",
         "WaitingForHost": "ホストを待っています…",
         "Yes": "はい",
         "yourEntireScreen": "あなたの画面全体"
     },
     "dialOut": {
-        "statusMessage": "は現在 {{status}} です"
+        "statusMessage": "は現在{{status}}です"
+    },
+    "documentSharing": {
+        "title": "共有ドキュメント"
+    },
+    "e2ee": {
+        "labelToolTip": "このミーティングの音声と動画通信は端末間で暗号化されています"
     },
     "feedback": {
         "average": "普通",
         "bad": "悪い",
         "detailsLabel": "それについてもっと教えてください。",
         "good": "良い",
-        "rateExperience": "ビデオ通話を評価してください",
+        "rateExperience": "ミーティングを評価",
         "veryBad": "とても悪い",
         "veryGood": "とても良い"
     },
     "incomingCall": {
-        "answer": "",
-        "audioCallTitle": "",
-        "decline": "却下",
-        "productLabel": "",
-        "videoCallTitle": "ビデオ通話着信"
+        "answer": "応答",
+        "audioCallTitle": "着信",
+        "decline": "拒否",
+        "productLabel": "Jitsi Meetから",
+        "videoCallTitle": "動画通話着信"
     },
     "info": {
-        "accessibilityLabel": "情報を表示する",
-        "addPassword": "",
-        "cancelPassword": "",
+        "accessibilityLabel": "情報を表示",
+        "addPassword": "$t(lockRoomPassword)を設定",
+        "cancelPassword": "$t(lockRoomPassword)を解除",
         "conferenceURL": "リンク：",
         "country": "国",
-        "dialANumber": "",
-        "dialInConferenceID": "PIN:",
+        "dialANumber": "ミーティングに参加するには、これらの番号のいずれかに電話をかけて、PINを入力してください。",
+        "dialInConferenceID": "PIN：",
         "dialInNotSupported": "申し訳ありませんが、現在ダイヤルインはサポートされていません。",
-        "dialInNumber": "ダイアルイン：",
-        "dialInSummaryError": "",
-        "dialInTollFree": "",
-        "genericError": "おっと、何かが間違っていました。",
-        "inviteLiveStream": "この会議のライブストリームを表示するには、このリンクをクリックしてください：{{url}}",
-        "invitePhone": "",
-        "invitePhoneAlternatives": "",
-        "inviteURLFirstPartGeneral": "",
-        "inviteURLFirstPartPersonal": "",
-        "inviteURLSecondPart": "",
+        "dialInNumber": "ダイヤルイン：",
+        "dialInSummaryError": "ダイヤルイン情報を取得できませんでした。後でもう一度お試しください。",
+        "dialInTollFree": "料金無料",
+        "genericError": "おっと、問題が発生しました。",
+        "inviteLiveStream": "このミーティングのライブストリームを表示するには、このリンクをクリックしてください：{{url}}",
+        "invitePhone": "電話で参加するには、これをタップしてください：{{number}},,{{conferenceID}}#\n",
+        "invitePhoneAlternatives": "別のダイヤルイン番号をお探しですか？\nミーティングのダイヤルイン番号をご参照ください：{{url}}\n\n\nルーム電話経由でダイヤルインしている場合は、オーディオに接続せずに参加してください：{{silentUrl}}",
```

Comment:
original: https://github.com/jitsi/jitsi-meet/blob/6773aed67f08879333f16118c1953c485b443094/lang/main.json#L351

The translation is not perfect as context is unclear: needs more info for `If also dialing-in through a room phone, join without connecting to audio`

Reply:
Context: https://github.com/jitsi/jitsi-meet/issues/6403#issue-607355132